ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/flow-proxy-form-dialog/gio-ps-flow-proxy-form-dialog.component.ts

```diff
@@ -95,10 +94,16 @@ export class GioPolicyStudioFlowProxyFormDialogComponent {
   }
 
   public tagValidationHook(tag: string, validationCb: (shouldAddTag: boolean) => void) {
-    validationCb(HttpMethods.map(m => `${m}`).includes(tag.toUpperCase()));
+    ttagValidationHook(tag, validationCb);
   }
 }
 
+const methods: HttpMethodVM[] = ['ALL', ...HttpMethods];
+
+const ttagValidationHook = (tag: string, validationCb: (shouldAddTag: boolean) => void) => {
```

Comment:
maybe not necessary ? as it's used only once no ?

Reply:
Optional. it's just that it's steange to see this function in one line. I'm thinking tagValidationHook only would be prettier. 
but it's style.. as you wish ;)